app/src/main/java/com/compscieddy/meetinthemiddle/GroupActivity.java

```diff
@@ -384,7 +404,41 @@ public void onClick(View v) {
         Etils.showToast(GroupActivity.this, "Invite Button Clicked");
         break;
 
-    }
+      case R.id.expand_chat_fab:
+
+        Display display = getWindowManager().getDefaultDisplay();
+        Point size = new Point();
+        display.getSize(size);
+        int height = size.y;
+
+        if (!expanded) {
+          mExpandButton.setImageResource(R.drawable.ic_expand_more_black_48dp);
+
+          RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams(
+              RelativeLayout.LayoutParams.MATCH_PARENT,
+              (int) (display.getHeight() * 0.75));
+          params.addRule(RelativeLayout.ALIGN_PARENT_BOTTOM);
+          mBottomSection.setLayoutParams(params);
 
+          Etils.showToast(GroupActivity.this, "Expand chat");
+
+          expanded = true;
+        } else {
+          mExpandButton.setImageResource(R.drawable.ic_expand_less_black_48dp);
+          Resources r = getResources();
+          float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 250, r.getDisplayMetrics());
+
+          RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams(
+              RelativeLayout.LayoutParams.MATCH_PARENT,
+              (int) px);
+          params.addRule(RelativeLayout.ALIGN_PARENT_BOTTOM);
```

Comment:
You can move this out of the if/else since it happens in both. Same with "mBottomSection.setLayoutParams(params)"